python/fusion_engine_client/messages/defs.py

```diff
@@ -237,7 +269,7 @@ def pack(self, buffer: bytes = None, offset: int = 0, payload: bytes = None, ret
             return self.calcsize()
 
     def unpack(self, buffer: bytes, offset: int = 0, validate_crc: bool = False,
-               warn_on_unrecognized: bool = True) -> int:
+               warn_on_unrecognized: bool = True, ignore_sync: bool = False) -> int:
```

Comment:
Are you actually using this? It seems a bit redundant to fix the preamble in the event unpack and also add this feature.

Reply:
Yes currently, but not sure we need it anymore. I added this before deciding to do the preamble fixup instead. I can remove it.